sdk/storage/storage-blob/samples/javascript/azureAdAuth.js

```diff
@@ -2,13 +2,14 @@
  Setup: Enter your Azure Active Directory credentials as described in main()
 */
 
-const { BlobServiceClient } = require("../.."); // Change to "@azure/storage-blob" in your package
-const { DefaultAzureCredential } = require("@azure/identity");
+import { BlobServiceClient } from "../../src"; // Change to "@azure/storage-blob" in your package
+import { DefaultAzureCredential } from "@azure/identity";
```

Comment:
Is this syntax supported in Node 8?

Reply:
No, it's not. Was just updating this in all the js files to `require` syntax.